technologies/app/saagie-hf-modelserver-textclf/saagie-hf-modelserver-textclf-0.1/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim
+ARG PYTHON_VERSION=3.9
+
+RUN pip install --upgrade pip
+RUN pip install numpy pandas transformers==4.16.2 pyarrow
+RUN pip3 install torch torchvision torchaudio transformers==4.16.2
+RUN pip install plotly dash==1.13.3 dash-bootstrap-components==0.13.1 Werkzeug==2.0.3
```

Comment:
all installation of python's librairies could be in the same pip3 install ? or you can put a requirements.txt and do something like: 

```
COPY requirements.txt ./
RUN pip install -r requirements.txt
```